### DISCUSSION TITLE
Começando na programação pelo JS

### DISCUSSION THREAD

**Discussion by deiviocoelho** [frontendbr/forum#2131] [Perguntas e Respostas]
Bom dia, estou começando os estudos na área de Ti, atualmente fazendo um bootcamp sobre lógica de programaçao (e um pouquinho de front end) da Digital Inovation One. Gostei bastante da área de front, e do que comecei a ver sobre JS em si (apesar de q estou aberto a conhecer a fullstack como um todo), e pesquisando aqui no fórum vi muita gente falando sobre a Rockseats e queria uma dica a respeito de curso introdutório deles ao Javascript...por qual começar?

**Comment by nazarepiedady** [comment]
Se pretendes ser desenvolvedor web, não deverias começar pelo código mas sim pela estrutura e funcionamento, com isso que dizer estudar assuntos como: 

* como funciona a internet?
* o que é a web?
* por quê a web existe?
* quais são os principais protocolos de comunicação da internet? 
* quais são os protocolos de comunicação usados na web?

e depois partir para os estudos de: HTML, CSS e só depois JavaScript.
